Configuration/HLT/python/autoCondHLT.py

```diff
@@ -1,5 +1,7 @@
 # https://cms-conddb.cern.ch/browser/#search
 
+snapshotTime = "9999-12-31 23:59:59.000"
```

Comment:
@fwyzard I have the same commit that you describe in https://github.com/cms-sw/cmssw/pull/12095#discussion_r43390240